ts/utils/guards.ts

```diff
@@ -1,6 +1,24 @@
+import { Wallet } from "../types/pagopa";
+
 /**
  * A type guard that filters out undefined and null from a type T
  */
 export function isDefined<T, O extends NonNullable<T>>(v: T): v is O {
   return v !== null && v !== undefined;
 }
+
+export const isExpiredCard = (w: Wallet) => {
```

Comment:
Can't be in this file because it is not a guard.

Reply:
ok, I'll move to a new utility file "checks.ts"

Reply:
I think "wallet.ts" is better